docs/guide/profiling-test-performance.md

```diff
@@ -11,16 +11,15 @@ When you run Vitest it reports multiple time metrics of your tests:
 > Test Files  1 passed (1)
 >      Tests  1 passed (1)
 >   Start at  09:32:53
->   Duration  4.80s (transform 44ms, setup 0ms, collect 35ms, tests 4.52s, environment 0ms, prepare 81ms)
+>   Duration  4.80s (transform 44ms, setup 0ms, import 35ms, tests 4.52s, environment 0ms, prepare 81ms)
 >   # Time metrics ^^
 > ```
 
 - Transform: How much time was spent transforming the files. See [File Transform](#file-transform).
 - Setup: Time spent for running the [`setupFiles`](/config/#setupfiles) files.
-- Collect: Time spent for collecting all tests in the test files. This includes the time it took to import all file dependencies.
+- Import: Time it took to import your test file. This also includes the time spent collecting all tests.
```

Comment:
I liked "all dependencies" wording of previous docs. I think "import test file" alone can feel un-intuitive. Can we keep that part like this?

```suggestion
- Import: Time it took to import your test file and all dependencies. This also includes the time spent collecting all tests.
```